### DISCUSSION TITLE
How to close toastMessage

### DISCUSSION THREAD

**Discussion by ghostgeek** [vuejs/pinia#1303] [Help and Questions] [answered]
`
🍍 "main" store installed 🆕
`
In my program, I want to close the 'toastMessage'

**Comment by posva** [comment, marked as answer]
Check the settings in the devtools to not show them:

![Screenshot 2022-05-19 at 09 33 07](https://user-images.githubusercontent.com/664177/169237619-d1f73a57-b0ff-400a-92aa-b953a84ee1c7.png)
s

**Reply by ghostgeek** [reply]
Thanks!
